polyply/data/parmbsc1/citations.bib

```diff
@@ -8,3 +8,11 @@ @article{parmbsc1
   year={2016},
   publisher={Nature Publishing Group}
 }
+
+@article{polyply,
+  title={Polyply; a python suite for facilitating simulations of (bio-) macromolecules and nanomaterials},
+  author={Grunewald, Fabian and Alessandri, Riccardo and Kroon, Peter C and Monticelli, Luca and Souza, Paulo CT and Marrink, Siewert J},
+  journal={Nature Communications},
```

Comment:
following https://www.nature.com/articles/s41467-021-27627-4, you can already add: 
```
volume={13},
pages={68},
```